CHANGELOG.md

```diff
@@ -4,7 +4,7 @@ The `@algolia/client-recommendation` is now deprecated, please use `@algolia/cli
 
 
 
-
+* feat(recommend): support multiple queries (#1283) ([5cf64dd](https://github.com/algolia/algoliasearch-client-javascript/commit/5cf64dd))
```

Comment:
this doesn't really need an entry, as it wasn't released without it
```suggestion
```

Reply:
See e03ceccb2b96b92f23a591dc01b79c940bde3e3e.